pegasus/sites.v3/code.org/public/educate/puzzlechallenge.md

```diff
@@ -1,6 +1,6 @@
 ---
```

Comment:
looks like there are JS errors for this page and the others that are just scripts. not sure what that's about 
<img width="1103" alt="screen shot 2017-04-03 at 1 45 12 pm" src="https://cloud.githubusercontent.com/assets/12300669/24631054/d9bf1618-1873-11e7-92d2-dcd324b3f0ba.png">